src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -264,11 +271,25 @@ export default function ConfirmedAppointmentDetailsPage() {
                     className="far fa-calendar vads-u-margin-right--1"
                   />
                   <AddToCalendar
-                    summary={`${header}`}
-                    description={calendarDescription}
-                    location={
-                      isPhone ? 'Phone call' : formatFacilityAddress(facility)
+                    summary={
```

Comment:
Why doesn't the code in here use data from `selectCalendarData`?

Reply:
Done.